installer/mod.ts

```diff
@@ -241,6 +241,11 @@ export async function install(
     }
   }
 
+  // if install local module
+  if (!/^https?:\/\//.test(moduleUrl)) {
```

Comment:
Yikes, this should have been merged in #499

Reply:
I miss it. :)

Reply:
I will add the relevant tests for the local module.